src/main/groovy/com/github/lkishalmi/gradle/gatling/GatlingRunTask.groovy

```diff
@@ -16,8 +18,12 @@ class GatlingRunTask extends JavaExec {
         classpath = project.configurations.gatlingRuntime
 
         args "-m"
-        if (project.sourceSets.gatling.output.hasProperty("classesDirs")) {
-            args "-bf", "${project.sourceSets.gatling.output.classesDirs[1]}" //TODO remove this hacky way of getting single output directory
+        if(GradleVersion.current() >= GradleVersion.version('4.0')) {
```

Comment:
It least at this line there's missing space between if and opening brace. Didn't looked on other places, though. Just format changed lines with default IDEA settings. That's will be OK.